pkgs/by-name/je/jellyfin-ffmpeg/package.nix

```diff
@@ -1,43 +1,112 @@
-{ ffmpeg_7-full
-, fetchFromGitHub
-, fetchpatch
-, lib
+{
+  ffmpeg,
+  fetchFromGitHub,
+  lib,
 }:
 
 let
   version = "7.0.2-5";
 in
 
-(ffmpeg_7-full.override {
+(ffmpeg.override {
   inherit version; # Important! This sets the ABI.
   source = fetchFromGitHub {
     owner = "jellyfin";
     repo = "jellyfin-ffmpeg";
     rev = "v${version}";
     hash = "sha256-cqyXQNx65eLEumOoSCucNpAqShMhiPqzsKc/GjKKQOA=";
   };
-}).overrideAttrs (old: {
-  pname = "jellyfin-ffmpeg";
-
-  configureFlags = old.configureFlags ++ [
-    "--extra-version=Jellyfin"
-    "--disable-ptx-compression" # https://github.com/jellyfin/jellyfin/issues/7944#issuecomment-1156880067
-  ];
-
-  postPatch = ''
-    for file in $(cat debian/patches/series); do
-      patch -p1 < debian/patches/$file
-    done
-
-    ${old.postPatch or ""}
-  '';
-
-  meta = {
-    inherit (old.meta) license mainProgram;
-    changelog = "https://github.com/jellyfin/jellyfin-ffmpeg/releases/tag/v${version}";
-    description = "${old.meta.description} (Jellyfin fork)";
-    homepage = "https://github.com/jellyfin/jellyfin-ffmpeg";
-    maintainers = with lib.maintainers; [ justinas ];
-    pkgConfigModules = [ "libavutil" ];
-  };
-})
+
+  ffmpegVariant = "barebones";
```

Comment:
Found a (silent) problem with using `barebones` here: Jellyfin uses the `ffprobe` binary to analyze media, which as of this commit is included in `headless`, but not `barebones`.

```
System.ComponentModel.Win32Exception (2): An error occurred trying to start process '/nix/store/0a2kih298ca607i07bd10lcrkfda9qm1-jellyfin-ffmpeg-7.0.2-5-bin/bin/ffprobe' with working directory '/var/lib/jellyfin'. No such file or directory
```

Let's set `buildFfprobe = true;` for `jellyfin-ffmpeg`.